js/index.js

```diff
@@ -0,0 +1,20 @@
+const today = new Date();
+const thisYear = today.getFullYear();
+
+const footer = document.querySelector('footer');
+
+const copyright = document.createElement('p');
+
+copyright.innerHTML = `Diana Carachure ${thisYear}`;
+
+footer.appendChild(copyright);
+
+const skills = ['JavaScript', 'HTML', 'CSS'];
+const skillsSection = document.getElementById('skills');
+const skillsList = skillsSection.querySelector('ul'); //replace document with skillsSection -only "looking" in that section
+
+for (let i = 0; i < skills; i++){
```

Comment:
So close! You need to compare `i` against `skills.length`, not the entire array `skills` 😄